core/src/test/java/com/linecorp/armeria/common/util/AbstractUnwrappableTest.java

```diff
@@ -35,6 +35,16 @@ void testUnwrap() {
         assertThat(qux.unwrap().unwrap()).isSameAs(foo);
     }
 
+    @Test
+    void testUnwrapAll() {
```

Comment:
Naming candidates I could think of are

1. `Unwrappable#root`
2. `Unwrappable#unwrapAll`
3. `Unwrappable#asRoot`

Reply:
It seems like `Unwrappable#unwrap(boolean)` is the least invasive and makes sense.
Credit to @ikhoon 👍 🙇